### PR TITLE
Feature/modify payment

### PR DESCRIPTION
**Pull request by NESOY**
## Modify Payment
- Can Change Payment Participation, Amount, Name
- Can't Modify Payment in Not logged-in

### CODE REVIEW

File: src/main/java/us/controller/MeetingController.java [hunk 1]
```diff
@@ -145,6 +145,25 @@ public String addPayment(@PathVariable int id, Payment payment) {
         updateMoneyToSend(addPayment);
         return "redirect:/meetings/" + id;
     }
+    @PostMapping(value = "/{id}/payment/{paymentId}/update")
```
Comment (sim4858) [inline src/main/java/us/controller/MeetingController.java:148]: I wonder about how be shown this request URL.

File: src/main/java/us/model/Payment.java [hunk 2]
```diff
@@ -65,4 +65,15 @@ public void setAmount(int amount) {
     public boolean isOwner(String ssoId) {
         return this.ssoId.equals(ssoId);
     }
+
+    @Override
+    public String toString() {
```
Comment (sim4858) [inline src/main/java/us/model/Payment.java]: Are you sure this method?

File: src/main/java/us/controller/MeetingController.java (hunk 1 above)
Comment (NESOY) [reply src/main/java/us/controller/MeetingController.java:148]: This URL is not showing page.
Just Handling Updating Payment from `<form>` tag

File: src/main/resources/static/js/btn.js [hunk 3]
```diff
@@ -10,4 +10,21 @@ $(window).click(function (event) {
...
     }
-});
\ No newline at end of file
+});
+
+$("#update_payment_dialog").on('show.bs.modal', function (event) {
+    var payment = $(event.relatedTarget);
+    var payer = payment.attr("name");
+    var paymentId = payment.attr("id");
+    var paymentName = payment.attr("value");
```
Comment (sim4858) [inline src/main/resources/static/js/btn.js]: I confuse of this elements.
We should decide one either name or value.

**Review by cjh5414 — APPROVED** [review]

**Pull request merged**